### ISSUE TITLE
Start command for activity

### ISSUE THREAD

**Issue by glsf91** [bwssytems/restful-harmony#13]
Is it also possible to start a command for an activity instead of a device?

**Comment by bwssytems** [comment]
Yes, it is in the readme here https://github.com/bwssytems/restful-harmony#start-an-activity

**Comment by glsf91** [comment]
That is only starting an activity. I ment starting a command belonging to an activity.
Like here https://github.com/sushilks/harmonyHubCLI.  Trigger a command within an activity

**Comment by bwssytems** [comment]
Yes, the are listed inside the activity structure and it lists the assigned buttons with the deviceId and button name that you can call with the button press command https://github.com/bwssytems/restful-harmony/blob/master/README.md#press-a-button-on-a-given-device
